content/reference/apis/standard.md

```diff
@@ -50,3 +50,5 @@ Both TextEncoder and TextDecoder support UTF-8 encoding/decoding.
 The URL API supports urls conforming to http and https schemes.
 
 [Go to the docs](https://developer.mozilla.org/en-US/docs/Web/API/URL)
+
+_Note: Our URL implementation differs slightly from standards. To use an external implementation, you can [shim the URL class using webpack](/tooling/wrangler/webpack/#shimming-globals)._
```

Comment:
The Workers' Runtime's URL class behavior differs from the URL Spec documented above. If you'd like to use another URL implementation you ... etc etc